### PR TITLE
[Wait for #1069] ini interpreter deserialize

### PR DESCRIPTION
**Pull request by zhoonit**
- **#1069** [Compiler/skeleton] Set up compiler source files
- **#1069** [Compiler] Add concept of the compiler
- **#1069** [Compiler] Declare interface
- **#1069** [Compiler] Add Graph Representation
- **#1069** [Compiler] Alternate GraphRepresentation
- [Ini] Implement ini deserializer
```
This patch implements ini deserializer as well as changing the interface
signature to get filepath.

Many NYI parts will be implemented soon.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>
```
- [IniInterpreter] Add section2layer
```
This patch add plain `section2layer`.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>
```
- [IniInterpreter] Add ini backbone deserialization
```
**Changes proposed in this PR:**
- Add ini backbone deserialization support

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>
```
- [IniInterpreter] Add test setup
```
This patch add a test setup to test interpreter.

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: Jihoon Lee <jhoon.it.lee@samsung.com>
```

### CODE REVIEW

File: nntrainer/compiler/ini_interpreter.cpp [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+#include <parse_util.h>
+#include <util_func.h>
+
+#include <nnstreamer_layer.h>
+#include <tflite_layer.h>
+
+static constexpr const char *FUNC_TAG = "[IniInterpreter] ";
+
+static constexpr const char *UNKNOWN_STR = "Unknown";
+static constexpr const char *NONE_STR = "NONE";
```
Comment (dongju-chae) [inline nntrainer/compiler/ini_interpreter.cpp:29]: upper cases only here ?

**Comment by zhoonit** [comment]
I was asked to try split PRs. This will be reuploaded spliting the commit size

File: nntrainer/compiler/ini_interpreter.cpp (hunk 1 above)
Comment (zhoonit) [reply nntrainer/compiler/ini_interpreter.cpp:29]: Thank you for the review, I'll update this :)

Comment (zhoonit) [reply nntrainer/compiler/ini_interpreter.cpp:29]: Updated here https://github.com/nnstreamer/nntrainer/pull/1078/files . Thank you 👍

**Pull request closed without merging**